build/three.js

```diff
@@ -325,19 +325,27 @@
 
 			// http://stackoverflow.com/questions/105034/how-to-create-a-guid-uuid-in-javascript/21963136#21963136
 
-			var lut = []; for (var i=0; i<256; i++) { lut[i] = (i<16?'0':'')+(i).toString(16).toUpperCase(); }
+			var lut = [];
+
+			for ( var i = 0; i < 256; i ++ ) {
+
+				lut[ i ] = ( i < 16 ? '0' : '' ) + ( i ).toString( 16 ).toUpperCase();
```

Comment:
Unrelated to this PR, but this caught my attention.

Why are we calling `toUpperCase()` 256 times instead of 1 time for the final result when the UUID is generated? This looks like a silly performance issue.

Reply:
>Unrelated to this PR

It is unrelated. Please file a new issue if you feel it is warranted.